xml/src/test/java/org/ehcache/xml/service/DefaultCacheLoaderWriterConfigurationParserTest.java

```diff
@@ -64,9 +57,9 @@ public void unparseServiceConfiguration() {
   public void unparseServiceConfigurationWithInstance() {
     TestCacheLoaderWriter testCacheLoaderWriter = new TestCacheLoaderWriter();
     CacheConfiguration<?, ?> cacheConfig =
-      buildCacheConfigWithServiceConfig(new DefaultCacheLoaderWriterConfiguration(testCacheLoaderWriter));
+      newCacheConfigurationBuilder(Object.class, Object.class, heap(10)).add(new DefaultCacheLoaderWriterConfiguration(testCacheLoaderWriter)).build();
     assertThatExceptionOfType(XmlConfigurationException.class).isThrownBy(() ->
-      parser.unparseServiceConfiguration(cacheConfig, new CacheType()))
+      new DefaultCacheLoaderWriterConfigurationParser().unparseServiceConfiguration(cacheConfig, new CacheType()))
       .withMessage("%s", "XML translation for instance based intialization for " +
```

Comment:
Would it be awful to correct the typo ("intialization" -> "initialization") in this exception message?

Reply:
Fixed (and in an independent commit).